### PR TITLE
changes to preserve order of homepage images :ok_hand:

### PR DESCRIPTION
**Pull request by frimpongopoku**
Related Ticket #547
Matching changes on frontend side as well here: https://github.com/massenergize/frontend-admin/pull/657 

### NEW MODEL 
 * ImageSequence model 

So you would have to run some light migrations.  

Now, we will be able to preserve the image sequence for the homepage images. And for all future scenarios that wld somehow need us to preserve image sequence, we cld just link to this model as well. 

[See demo on frontend PR](https://github.com/massenergize/frontend-admin/pull/657 )

### CODE REVIEW

File: src/database/models.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
-import dataclasses
 import datetime
 import json
+from tkinter.tix import IMAGE
```
Comment (BradHN1) [inline src/database/models.py:3]: Remove this tkinter import

**Review by Davidjwkinghorn — APPROVED** [review]
LGTM

**Comment by Davidjwkinghorn** [comment]
When I merged into canary these changes prevented me from loading content from the frontend-portal and the admin-portal. I got "community does not exist". I've rolled back these changes on canary and redeployed on dev and canary for now.

**Comment by frimpongopoku** [comment]
Community does not exist error usually happens when your frontend is not communicating with it's right counterpart. Like local is talking to dev db, or canary is talking to  dev db, or something like that.....

> When I merged into canary these changes prevented me from loading content from the frontend-portal and the admin-portal. I got "community does not exist". I've rolled back these changes on canary and redeployed on dev and canary for now.

**Comment by Davidjwkinghorn** [comment]
That's what I thought too, but I've triple checked that everything is set to the correct settings and I'm still getting this issue.
Screen Shot 2022-08-19 at 12 42 17 PM

**Pull request merged**